$MirOS: ports/emulators/qemu/patches/patch-block-vpc_c,v 1.2 2005/11/15 12:25:08 tg Exp $
--- block-vpc.c.orig	Tue Nov 15 11:52:07 2005
+++ block-vpc.c	Tue Nov 15 11:52:50 2005
@@ -239,4 +239,8 @@ BlockDriver bdrv_vpc = {
     vpc_read,
     NULL,
     vpc_close,
+    NULL,
+    NULL,
+    NULL,
+    NULL
 };
